VirtoCommerce.AzureSearchModule.Data/AzureSearchProvider.cs

```diff
@@ -338,7 +338,7 @@ protected virtual Index CreateIndexDefinition(string indexName, IList<Field> pro
                     {
                         Name = ContentAnalyzerName,
                         Tokenizer = TokenizerName.Standard,
-                        TokenFilters = new[] { TokenFilterName.Lowercase, GetTokenFilterName() }
+                        TokenFilters = new[] { TokenFilterName.Lowercase, GetTokenFilterName(), TokenFilterName.AsciiFolding  }
```

Comment:
@artem-dudarev Need your expert opinion on the following question: why we are using different analyzers for search and index?
https://github.com/VirtoCommerce/vc-module-azure-search/blob/62ec8d6c779c81b71df9c1a6083b5722878b6cdb/VirtoCommerce.AzureSearchModule.Data/AzureSearchProvider.cs#L230

I suspect that if we add such filter to the custom analyzer (which is IndexAnalyzer), and StandardLucene will be used as a SearchAnalyzer, we could have some troubles when non-ascii symbols will be used in queries.

Could you confirm we should not modify custom analyzer filters that way?
And also if having different analyzers for Index and Search is necessary (MS [recommends](https://docs.microsoft.com/en-us/azure/search/search-analyzers#one-analyzer-for-read-write-unless-you-have-specific-requirements) them to be identical in regular scenario)?